src/components/Navigation.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navigation() {
+  return (
+    <nav>
+      <ul>
+        <li>
+          <Link to="/">Home</Link>
+        </li>
+        <li>
+          <Link to="/categories">Categories</Link>
```

Comment:
- Optional
Instead of using `Link` component, I would recommend using `NavLink` component. You can read difference between the two [here](https://stackoverflow.com/questions/47338077/react-router-v4-navlink-vs-link-benefits).